products/tarantool.md

```diff
@@ -20,7 +20,7 @@ releases:
 
     releaseDate: 2022-05-22
 -   releaseCycle: "2.8"
-    eol: false
+    eol: true
```

Comment:
eol: 2022-04-25

Reply:
Ouch, sorry. Fixed.

Reply:
> Ouch, sorry. Fixed.

No need to sorry , it is completely okey to use true but having date will also add a little more info for users.
Thanks you for your contribution 🎉 🥇 👍 💯